### PR TITLE
Patch mk-ca-bundle to make ca-certificates reproducible regardless of current date

### PR DESCRIPTION
**Pull request by eduardosm**
Fixes https://github.com/fosslinux/live-bootstrap/issues/220.

Supersedes https://github.com/fosslinux/live-bootstrap/pull/221.

Based on the tips from https://github.com/fosslinux/live-bootstrap/pull/221#issuecomment-1374610121.

Also updated ca-certificates package to the latest version (3.88.1)

### CODE REVIEW

**Comment by fosslinux** [comment]
Could you please squash the first and last commit, and make the header say have your copyright (in SPDX-CopyrightHeader) instead of mine?

**Comment by eduardosm** [comment]
> Could you please squash the first and last commit, and make the header say have your copyright (in SPDX-CopyrightHeader) instead of mine?

Done

**Review by fosslinux — CHANGES_REQUESTED** [review]
Only one small problem remaining

File: sysc/ca-certificates-3.88.1/sources [hunk 1]
```diff
@@ -0,0 +1 @@
+https://ftp.mozilla.org/pub/security/nss/releases/NSS_3_88_1_RTM/src/nss-3.88.1.tar.gz 27d243edf87d1cf1bb9c861f03d387e0e9230ce5017f4308c941f558b54b3496
```
[the remark this answers is not in the document]
Comment (eduardosm) [reply sysc/ca-certificates-3.88.1/sources]: Makes sense, I hadn't thought of that. Changed and squashed into the second commit.

**Review by stikonas — APPROVED** [review]
Looks good.

Do we want to keep the second build of curl unpatched? I guess we won't have to do second build of ca-certificates...

**Comment by stikonas** [comment]
One final thing: could you please add "Fixes #220" to the commit message of the last commit (on a separate line)

**Comment by eduardosm** [comment]
> One final thing: could you please add "Fixes #220" to the commit message of the last commit (on a separate line)

Done

> Do we want to keep the second build of curl unpatched? I guess we won't have to do second build of ca-certificates...

I believe we are not even installing `mk-ca-bundle` in the second build.

The first build installs it with a dedicated `install` invocation.

https://github.com/fosslinux/live-bootstrap/blob/c0ea746ae534be8b07b32c353b59346c6beb1026/sysa/curl-7.83.0/curl-7.83.0.sh#L31-L34

The second build does not have a custom `src_install` defined.

**Review by stikonas — APPROVED** [review]
Will merge once CI finishes.

**Pull request merged**